buku

```diff
@@ -618,11 +619,15 @@ class BukuDb:
             return
 
         if not self.json:
-            prompt(results, self.noninteractive)
+            if delete is False:
+                prompt_open(results, self.noninteractive)
```

Comment:
I see. But delete is a different operation and one can pass one of the flags -S, -s, --st, then accidentally touch 'd' (delete) instead of 'o' (open)